bitnami/airflow/README.md

```diff
@@ -95,9 +95,10 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `airflow.gitSyncContainer.args`                | Override default container args (useful when using custom images)                                                                                                                                                              | `nil`                                                        |
 | `airflow.gitSyncContainer.extraVolumeMounts`   | Array of extra volume mounts to be added (evaluated as template). Normally used with `extraVolumes`.                                                                                                                           | `nil`                                                        |
 | `airflow.cloneDagFilesFromGit.enabled`         | Enable in order to download DAG files from git repository.                                                                                                                                                                     | `false`                                                      |
-| `airflow.cloneDagFilesFromGit.repository`      | Repository where download DAG files from                                                                                                                                                                                       | `nil`                                                        |
-| `airflow.cloneDagFilesFromGit.branch`          | Branch from repository to checkout                                                                                                                                                                                             | `nil`                                                        |
-| `airflow.cloneDagFilesFromGit.path`            | Path to a folder in the repository containing DAGs. If not set, all DAGS from the repo are loaded.                                                                                                                             | `nil`                                                        |
+| `airflow.cloneDagFilesFromGit.repositories[0].repository` | Repository where download DAG files from                                             | `nil`                                                        |
+| `airflow.cloneDagFilesFromGit.repositories[0].name` | An unique identifier for repository, must be unique for each repository                    | `nil`                                                        |
+| `airflow.cloneDagFilesFromGit.repositories[0].branch`     | Branch from repository to checkout                                                   | `nil`                                                        |
+| `airflow.cloneDagFilesFromGit.repositories[0].path`       | Path to a folder in the repository containing DAGs. If not set, all DAGS from the repo are loaded.   | `nil`                                                        |
```

Comment:
Could you make sure the table is properly alligned? So it is easier to read when editing it